src/Symfony/Component/PropertyInfo/Tests/Fixtures/Php74Dummy.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Symfony\Component\PropertyInfo\Tests\Fixtures;
+
+/**
+ * Class Php74Dummy
+ *
+ * @author Tales Santos <tales.augusto.santos@gmail.com>
+ */
+class Php74Dummy
+{
+    public int $int;
+    public ?string $string;
+    public Dummy $dummy;
+    public ?Dummy $optionalDummy;
```

Comment:
It looks like you forgot about "callable" property

Reply:
Actually, this test case should be removed because declaring `callable` for a property is not allowed. https://wiki.php.net/rfc/typed_properties_v2